### ISSUE TITLE
Maybe put a burb somewhere about screen layouts for online workshops

### ISSUE THREAD

**Issue by chendaniely** [carpentries/instructor-training#1294]
I'm not sure where exactly this would go, but since online workshops are going to be more of a norm, screen layouts and instructor screen sharing size may be something that goes under extraneous load in the "memory and cognitive load" chapter or maybe under the inclusion chapter since not all learners (and instructors) will have access to multiple screens.

This was the link to the blog post that shows what screen layouts would look like with a single screen in mind: Online workshop logistics and screen layouts: https://carpentries.org/blog/2020/06/online-workshop-logistics-and_screen-layouts/

<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this contribution is for instructor training, please email the link to this contribution to
instructor.training@carpentries.org so we can record your progress. You've completed your contribution
step for instructor checkout by submitting this contribution!

If this issue is about a specific episode within a lesson, please provide its link or filename.

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution. Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact The Carpentries Team at team@carpentries.org.

You may delete these instructions from your comment.

\- The Carpentries
</details>

**Comment by hannahksmith** [comment]
As someone who worked on  a single screen from 2020 through most of 2021, I appreciate the mention of single screens! 

One thing that I, personally, found useful was a feature through Zoom that could be access by going through Share Screen>Advanced>Portion of Screen. This allowed you to select a portion of the screen that would be shared with viewers - as opposed to selection one specifc window, this allowed you to toggle between windows for the viewer to see (as long as the windows were within the portion of the screen that was selected) while still allowing the instructor to have additional windows were only viewable to them (as long as they were outside of the selected portion of the screen).

With that being said, this solution may still be problematic for instructors with smaller screens. Any additional suggestions would be greatly appreciated!

**Comment by brownsarahm** [comment]
Do we point to online workshops formally anywhere? I think I share the overall page al ot in the q&a part on the day 2 morning (part3) but maybe this should be in the preparing to teach section?  Like where we mention [instructor notes](https://carpentries.github.io/instructor-training/18-preparation/index.html#review-the-instructor-notes)?

**Comment by JohnMulligan** [comment]
I agree with @hannahksmith's  specific suggestion above that the instructor screen usage recommendations be updated to include "portion of screen" sharing.

This, especially combined with the multiple-desktop tip (and a little planning), allows for a customizing what students see and the resolution it's delivered to them in according to the lesson's needs.

It also avoids the annoying effect where, when an instructor ends one share and starts another (say, choosing specific application windows) mid-lesson, the student's screen jumps back and forth and they then have to re-tile things on their end.

**Comment by karenword** [comment]
Thanks to all for these excellent ideas! We are currently in the process of figuring out where our advice about online workshops will live and how it can be incorporated into our standard IT curriculum. Tagging @sheraaronhurt for consideration in the bigger picture of that transition.

**Comment by sheraaronhurt** [comment]
Thank you so much everyone. I am working with the Workshop Administration Team to update online content. This information is very useful!

**Comment by annajiat** [comment]
Should the link be added to [helper and instructor checklists](https://docs.carpentries.org/topic_folders/hosts_instructors/hosts_instructors_checklist.html)

like
For online or hybrid workshop, check [how to manage logistics and screen layouts](https://carpentries.org/blog/2020/06/online-workshop-logistics-and_screen-layouts/)

**Comment by ndporter** [comment]
This feels like it belongs in the [bonus module](https://carpentries.github.io/instructor-training-bonus-modules/02-online-workshops-module-2.html) at a minimum. There could be a note in the `setup` pages on recommendations for screen layout for online workshops with one small screen (or a standard link somewhere that's easy to add to those pages in each lesson). But is it something to spend time on in the workshops themselves? I don't know.
